examples/sync-pass/src/sync-pass.rs

```diff
@@ -274,32 +272,27 @@ fn prompt_record_id(s: &LoginStore, action: &str) -> Result<Option<String>> {
     Ok(Some(index_to_id[input].as_str().into()))
 }
 
-fn open_database(db_path: &str) -> Result<LoginStore> {
-    let encryption_key = get_or_create_encryption_key()?;
-    let encdec = Arc::new(ManagedEncryptorDecryptor::new(Arc::new(
-        StaticKeyManager::new(encryption_key),
-    )));
-    let store = LoginStore::new(db_path, encdec)?;
-    Ok(store)
-}
+struct MyPrimaryPasswordAuthenticator {}
+impl PrimaryPasswordAuthenticator for MyPrimaryPasswordAuthenticator {
+    fn get_primary_password(&self) -> Result<String, LoginsApiError> {
+        let password = prompt_string("primary password").unwrap_or_default();
+        Ok(password.to_owned())
```

Comment:
a) Thanks for updating `sync-pass`, it's great to keep the example code up-to-date.
b) In this case, a sync method is actually perfect.  However, I think this could be could be converted to be a "fake-async" method without too much work.  Given that there's no await points, maybe pollster could even poll it to completion by itself.

Reply:
yes, it works with the async trait just the same way